wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss

```diff
@@ -271,7 +298,11 @@
 
   .cite__container {
     @extend %text-style-paragraph-small;
+    max-width: rem(500);
     color: $color-black-165;
+    @include breakpoint(large) {
+      max-width: calc(100% - 308px);
```

Comment:
Let's confirm with Christina that the citation should follow the width of the authors.

Reply:
Confirmed with her yesterday